<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="_2">
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Time Series - 24 hours</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="timeSlider">
      <property name="maximum">
       <number>23</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="air_hlayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>318</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>301</width>
           <height>1016</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>277</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Targets</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_air_generate">
               <property name="text">
                <string>Generate Objects</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="number">
                 <property name="minimum">
                  <number>2000</number>
                 </property>
                 <property name="maximum">
                  <number>6000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_36">
                 <property name="text">
                  <string>&lt;b&gt;Airplanes&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="airplanesSB">
                 <property name="maximum">
                  <number>6000</number>
                 </property>
                 <property name="value">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Max. Velocity (km/h)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MaxVel">
                 <property name="minimum">
                  <double>900.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>8000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>8000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Min. Velocity (km/h)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MinVel">
                 <property name="minimum">
                  <double>900.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>8000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Max. Acceleration (m/s)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MaxAccel">
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Min. Acceleration (m/s)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MinAccel"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Max. Altitude (km)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MaxAlt">
                 <property name="maximum">
                  <double>112.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Min. Altitude (km)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="MinAlt"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_39">
               <item>
                <widget class="QLabel" name="label_37">
                 <property name="text">
                  <string>&lt;b&gt;Helicopters&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="helicoptersSB">
                 <property name="maximum">
                  <number>6000</number>
                 </property>
                 <property name="value">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Rotation Angle (RA)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="RA">
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>RA Starting Time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="RAST">
                 <property name="maximum">
                  <double>86400.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_37">
               <item>
                <widget class="QLabel" name="label_38">
                 <property name="text">
                  <string>&lt;b&gt;Cars&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="carsSB">
                 <property name="maximum">
                  <number>6000</number>
                 </property>
                 <property name="value">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_38">
               <item>
                <widget class="QLabel" name="label_39">
                 <property name="text">
                  <string>&lt;b&gt;Ships&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="shipsSB">
                 <property name="maximum">
                  <number>6000</number>
                 </property>
                 <property name="value">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_32">
               <item>
                <widget class="QLabel" name="label_41">
                 <property name="text">
                  <string>Max. Depth (m)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox">
                 <property name="maximum">
                  <number>600</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_41">
               <item>
                <widget class="QLabel" name="label_42">
                 <property name="text">
                  <string>Min. Depth (m)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_4">
                 <property name="maximum">
                  <number>600</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Initial Instant (h)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="initTime">
                 <property name="maximum">
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_40">
               <item>
                <widget class="QLabel" name="label_40">
                 <property name="text">
                  <string>Life Time (h)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="lifeTime">
                 <property name="maximum">
                  <number>24</number>
                 </property>
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Moving Pattern</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="airMovingPattern">
                 <item>
                  <property name="text">
                   <string>Constant Velocity</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Constant Acceleration</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Constant Deceleration</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Pattern Switch Time (s)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxAirPST">
                 <property name="maximum">
                  <number>24</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="AddPattern">
               <property name="text">
                <string>Add Pattern</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="patternTable">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>200</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="trajectoryHLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>197</height>
         </size>
        </property>
        <property name="title">
         <string>Trajectory</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_35">
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>Object ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="objectsID"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_36">
             <item>
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>Object Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="objectType">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>27</height>
                </size>
               </property>
               <property name="documentTitle">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="plainText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_34">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Start Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="objectST">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>27</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_33">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>End Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="objectET">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>27</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
